draft/7/4d-projection-and-basic-hypercube-twists/index.md

```diff
@@ -0,0 +1,133 @@
+# Cubing Standard #7: 4D Projection and Basic Hypercube Twists (DRAFT)
+
+Status: `DRAFT`  
+Prescriptive or descriptive: descriptive (this standard attempts to describe existing conventions)
+
+This standard describes the basic moves for a NxNxNxN facet-turning hypercube.
+
+## 7.1 Polytope elements
+
+When possible, pieces are named according to the rank of the element of the polytope they correspond to. Let _n_ be the rank of the puzzle.
+
+| Rank of element | Name   |
+| --------------- | ------ |
+| 0               | Corner |
+| 1               | Edge   |
+| 2               | Face   |
+| 3               | Cell   |
+| _n_ - 3         | Peak   |
+| _n_ - 2         | Ridge  |
+| _n_ - 1         | Facet  |
+
+These terms are taken from [Polytope - Wikipedia](https://en.m.wikipedia.org/wiki/Polytope#Elements), with the exception of "corner" instead of "vertex."
+
+The terms "corner" and "edge" are preferred over "peak," "ridge," and "facet." For puzzles in 4 or more dimensions, the terms "peak", "ridge", and "facet" are preferred over "face" and "cell."
+
+For a 4D hypercube, the terms "corner," "edge," "ridge," and "facet" are preferred, although "cell" is also in common use.
+
+## 7.1 Facets
+
+A **facet** is the (N-1)-dimensional element of a polytope. A **cell** is 3-dimensional element. For a 4D polytope, "facet" and "cell" are interchangeable.
+
+There are 8 facets of a 4-dimensional hypercube:
+
+| Facet letter | Facet name | Vector     | Standard color |
+| ------------ | ---------- | ---------- | -------------- |
+| `R`          | Right      | positive X | Red            |
+| `L`          | Left       | negative X | Orange         |
+| `U`          | Up         | positive Y | White          |
+| `D`          | Down       | negative Y | Yellow         |
+| `F`          | Front      | positive Z | Front          |
+| `B`          | Back       | negative Z | Back           |
+| `O`          | Out        | positive W | Pink           |
+| `I`          | In         | negative W | Purple         |
+
+Note: historically, `K` ("kata") has been used instead of `I`, and `A` ("ana") or `T` ("top") have been used instead of `I`. It is recommended to avoid these because it is confusing which one corresponds to which cell. Furthermore, `A` conflicts with 5D notation (`A` for "anterior") and `T` has been reappropriated for general F2L notation (`T` for general "top").
```

Comment:
Hmm, this is the first I'm hearing of "general F2L notation" or of a modern use of `T`. Do you have a citation?

Reply:
I don't expect any software to understand `T` as referring to "top" because this is really just a tool for humans to talk about things at a high level, so there's no notational conflict. I'll remove that part of the note.